client/context.go

```diff
@@ -329,27 +336,32 @@ func GetFromFields(kr keyring.Keyring, from string, genOnly bool) (sdk.AccAddres
 		return addr, "", 0, nil
 	}
 
-	var info keyring.Info
+	ke := new(keyring.Record)
```

Comment:
maybe:

```suggestion
	kr := new(keyring.Record)
```

Reply:
Since `kr` variable is already declared in `GetFromFields`, I use `re` instead.